float64 lat
float64 long
float64 alt
bool obstacle_detected